Dart/src/com/jetbrains/lang/dart/ide/runner/server/DartCommandLineRunnerParameters.java

```diff
@@ -151,6 +154,22 @@ public VirtualFile getDartFileOrDirectory() throws RuntimeConfigurationError {
     return dartFile;
   }
 
+  @Nullable
+  public Module getModule(@NotNull Project project) {
```

Comment:
sounds good (this might have been from before I moved to using the global scope)